includes/Core/Assets/Script.php

```diff
@@ -53,15 +55,23 @@ public function __construct( $handle, array $args ) {
 	 * Registers the script.
 	 *
 	 * @since 1.0.0
+	 * @since n.e.x.t Adds $context parameter.
+	 *
+	 * @param Context $context Plugin context.
 	 */
-	public function register() {
+	public function register( Context $context ) {
 		if ( $this->args['fallback'] && wp_script_is( $this->handle, 'registered' ) ) {
 			return;
 		}
 
+		$src = $this->args['src'];
+		if ( class_exists( '\Google\Site_Kit\Core\Assets\Manifest' ) && isset( Manifest::$assets[ $this->get_handle() ] ) ) {
```

Comment:
We should probably expect this class to exist here, otherwise our plugin is broken.
```suggestion
		if ( isset( Manifest::$assets[ $this->handle ] ) ) {
```

Also, based on my other comment, if this is not found, let's try stripping the `googlesitekit-` prefix and finding it from that.

Reply:
Updated.